docs/ReleaseNotes.md

```diff
@@ -16,7 +16,7 @@ Constructors of the `RecordQueryUnionPlan` and `RecordQueryIntersectionPlan` hav
 // begin next release
 ### NEXT_RELEASE
 
-* **Bug fix** Fix 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Bug fix** The `SizeStatisticsCursorContinuation` must not access the continuation of its underlying cursor after it is exhausted [(Issue #656)](https://github.com/FoundationDB/fdb-record-layer/pull/657)
```

Comment:
```suggestion
* **Bug fix** The `SizeStatisticsCursorContinuation` must not access the continuation of its underlying cursor after it is exhausted [(Issue #656)](https://github.com/FoundationDB/fdb-record-layer/issues/656)
```

Hm, I'm not opposed necessarily to including PR information. I think we've generally included issue information (with the idea being that that should generally then also reference its fixing PR, though I've also seen that linking not work all that well, which is why I tend to mention that a PR fixes #656, etc.).